ocp_resources/storage_profile.py

```diff
@@ -4,4 +4,33 @@
 
 
 class StorageProfile(Resource):
+    """
+    StorageProfile Object
+
+    Doc:
+    https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md
+    """
+
     api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
+
+    @property
+    def claim_property_sets(self):
+        return self.instance.status.get("claimPropertySets")
```

Comment:
```suggestion
        return self.instance.status.get("claimPropertySets", {})
```

then you can remove `if self.claim_property_sets else None` from `first_claim_......`

Reply:
So it will produce a KeyError if fails?
y nice